apps/docusaurus/docs/standards/aptos-object.md

```diff
@@ -159,8 +159,8 @@ struct DeleteRefStore has key {
     delete_ref: DeleteRef,
 }
 
-public fun delete_liquidity_pool(liquidity_pool: Object<LiquidityPool>) {
-    let liquidity_pool_address = object::object_address(liquidity_pool);
+public fun delete_liquidity_pool(liquidity_pool: Object<LiquidityPool>) acquires LiquidityPool, DeleteRefStore {
+    let liquidity_pool_address = object::object_address(&liquidity_pool);
```

Comment:
Please make all these fixes also to apps/nextra/docs.  We're moving soon!

Reply:
> Please make all these fixes also to apps/nextra/docs. We're moving soon!

Done